test/WebJobs.Script.Tests/Configuration/ServerlessSecurityServiceOptionsTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Az.ServerlessSecurity.Platform;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ServerlessSecurityServiceOptionsTests : IDisposable
+    {
+        private string traceloggerFilename;
+        private string localFilePath;
+        private string initialValueServerlessSecurityLogConfig;
```

Comment:
C# naming convention for private variables starts with underscore. _initialValueServerlessSecurityLogConfig, _traceloggerFilename, _localFilePath

Reply:
👍